tmol/score/disulfide/disulfide_whole_pose_module.py

```diff
@@ -0,0 +1,75 @@
+import torch
+
+from tmol.score.disulfide.potentials.compiled import disulfide_pose_scores
+
+
+class DisulfideWholePoseScoringModule(torch.nn.Module):
+    def __init__(
+        self,
+        pose_stack_block_coord_offset,
+        pose_stack_block_types,
+        pose_stack_inter_block_connections,
+        bt_disulfide_conns,
+        bt_atom_downstream_of_conn,
+        global_params,
+    ):
+        super(DisulfideWholePoseScoringModule, self).__init__()
+
+        def _p(t):
+            return torch.nn.Parameter(t, requires_grad=False)
+
+        def _t(ts):
+            return tuple(map(lambda t: t.to(torch.float), ts))
+
+        self.pose_stack_block_coord_offset = _p(pose_stack_block_coord_offset)
+        self.pose_stack_block_types = _p(pose_stack_block_types)
+        self.pose_stack_inter_block_connections = _p(pose_stack_inter_block_connections)
+        self.bt_disulfide_conns = _p(bt_disulfide_conns)
+        self.bt_atom_downstream_of_conn = _p(bt_atom_downstream_of_conn)
+
+        self.global_params = _p(
```

Comment:
Something like this? https://github.com/uw-ipd/tmol/pull/254/commits/99023c48306a08c616d2f38c0071aae5c5ff2d37

It feels a little weird to be putting this in the packed block types, since it isn't a feature of the block types and this data won't change if the active block types change. We could consider putting it in DisulfideEnergyTerm::\_\_init\_\_ and have it done once right after it loads the params from the database.